src/universal/modules/meeting/components/AgendaCard/AgendaCard.js

```diff
@@ -0,0 +1,214 @@
+import React, {Component, PropTypes} from 'react';
+import look, {StyleSheet} from 'react-look';
+import {cashay} from 'cashay';
+import {Field, reduxForm, initialize, focus} from 'redux-form';
+import labels from 'universal/styles/theme/labels';
+import TayaAvatar from 'universal/styles/theme/images/avatars/taya-mueller-avatar.jpg';
+import OutcomeCardAssignMenuContainer
+  from 'universal/modules/teamDashboard/containers/OutcomeCardAssignMenu/OutcomeCardAssignMenuContainer';
+
+import OutcomeCard from 'universal/components/OutcomeCard/OutcomeCard';
+import OutcomeCardTextareaContainer from 'universal/modules/teamDashboard/containers/OutcomeCardTextarea/OutcomeCardTextareaContainer';
+import OutcomeCardFooter from 'universal/components/OutcomeCard/OutcomeCardFooter';
+import OutcomeCardStatusMenu from 'universal/components/OutcomeCard/OutcomeCardStatusMenu';
+import getOutcomeNames from 'universal/utils/getOutcomeNames';
+
+const OPEN_CONTENT_MENU = 'AgendaCard/openContentMenu';
+const OPEN_ASSIGN_MENU = 'AgendaCard/openAssignMenu';
+const OPEN_STATUS_MENU = 'AgendaCard/openStatusMenu';
+
+let styles = {};
+
+@reduxForm()
+@look
+export default class AgendaCard extends Component {
```

Comment:
Yeah, we've got some refactoring to do coming up, I figured it best to hash out all the cards & then we can simplify once we see me all lined up.